### PR TITLE
hint secp for jetting

### PR DESCRIPTION
**Pull request by BernardoDeLaPlaz**

### CODE REVIEW

File: sys/zuse.hoon [hunk 1]
```diff
@@ -3348,21 +3348,26 @@
       ==
     ::
     ++  secp
-      |=  [w=@ p=@ a=@ b=@ g=pont n=@]
+     ~/  %secp
+      |=  [w=@ p=@ a=@ b=@ g=pont n=@]  :: being passed in from above
       =/  p  ~(. fo p)
       =/  n  ~(. fo n)
+      ~%  %helper  ..$
```
Comment (Fang-) [inline sys/zuse.hoon]: Syntax error on the line below, I think you're missing a `~` here? ie `~%  %helper  ..$  ~`

Comment (BernardoDeLaPlaz) [reply sys/zuse.hoon]: on it.

**Comment by Fang-** [comment]
Ping @BernardoDeLaPlaz, crypto-additions is getting merged into the release candidate soon, can you push your fix for this, and any other changes you have made for the rest of the jets?

**Comment by joemfb** [comment]
@BernardoDeLaPlaz bumping this PR as well

File: sys/zuse.hoon (hunk 1 above)
Comment (BernardoDeLaPlaz) [reply sys/zuse.hoon]: and, 1 month after doing it, actually pushing it...

File: sys/zuse.hoon [hunk 2]
```diff
@@ -3348,21 +3348,26 @@
       ==
     ::
     ++  secp
-      |=  [w=@ p=@ a=@ b=@ g=pont n=@]
+     ~/  %secp
+      |=  [w=@ p=@ a=@ b=@ g=pont n=@]  :: being passed in from above
       =/  p  ~(. fo p)
       =/  n  ~(. fo n)
+      ~%  %helper  ..$ ~
```
Comment (joemfb) [inline sys/zuse.hoon]: I think you'll need one more space between `..$` and `~`.

**Comment by joemfb** [comment]
closing in favor of #837

**Pull request closed without merging**